spec/fixtures/puppetdb/ssl-setup.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
Is there a reason to have your own copy of this file?

I think you could just expect this to be in the upstream puppetdb image at `/ssl-setup.sh`, and call that.

Reply:
I think the only change was we hardcoded the value of `mycertname` to be `pdb` instead of using the `HOSTNAME` environment variable. I am testing out setting `HOSTNAME=pdb` in our dockerfile to see if we can avoid maintaining our own copy of ssl-setup.sh.

Reply:
Just setting the environment variable appears to work. This will allow us to utilize the script that ships with the image.

Reply:
Ah, didn't notice that difference.

A few thoughts come to mind:

* The hardcoded ssl fixture files are expected to be named after the hostname, since things will try and use `$HOSTNAME` or `puppet config print certname` to find them. If you have fixtures named `certs/puppetdb.pem` then ssl-setup.sh would not find them if it was using HOSTNAME=pdb since it'll look for `certs/pdb.pem`
* If you do overwrite HOSTNAME, limit the scope as much as possible by putting it in front of the command that needs it, like `HOSTNAME=pdb /ssl-setup.sh`, versus at a global/higher scope
* We can just change the real ssl-setup.sh file in puppetdb so this hostname can be passed in